redcap_bridge/project_building.py

```diff
@@ -160,6 +166,17 @@ def combine_values(value1, value2):
     if output_file is not None:
         combined_df.to_csv(output_file, index=False)
 
+    # print number of customizing field
+    print("\nCustomizing", len(customization_df) , "fields\n", end='')
+
+    # print the field name
+    if(len(customization_df)) < 100:
+        customization_df = pd.read_csv(customization_csv, usecols=[0])
```

Comment:
I am not sure if re-reading the file again just for the print is a good idea when the information is in principle already loaded... can't you just use the index of the existing customization_df?